addons/bus/static/src/services/worker_service.js

```diff
@@ -81,12 +81,17 @@ export class WorkerService {
     /**
      * Send a message to the worker.
      *
-     * @param {String} action Action to be executed by the worker.
-     * @param {Object|undefined} data Data required for the action to be
-     * executed.
+     * @param {string} action Action to be executed by the worker.
+     * @param {Object} [options]
+     * @param {any} [options.data] Data required by the action.
+     * @param {boolean} [options.ensureWorker=false] Whether to start the worker
```

Comment:
Nitpick: would rename to `autostartWorker` so this is more intuitive of the impact of this options.

Reply:
Talked about it with seb. We decided it would be better to actually call `ensureWorkerStarted` if we really need to. It doesn't pollute the `send` method API, is easier to use/understand and it doesn't expose the worker service internal state.